### ISSUE TITLE
Dropdown does not work with iOS VoiceOver

### ISSUE THREAD

**Issue by haavardnk** [salim-lachdhaf/searchable_dropdown#519]
**Describe the bug**
When using dropdown multiselect, the popup window does not work with Apples VoiceOver. I have some users using this feature, and implementing this package has removed their possibility of filtering.

**To Reproduce**
1. Activate VoiceOver in iOS accesibility menu
2. Open dropdown multiselect with search
3. Try to select a line

**Expected behavior**
It is expected to be able to select the searchbar and each line in the dropdown, including the check box. The semantics should read the text from the selected line.

**Screenshots**
https://user-images.githubusercontent.com/29313194/206951992-adc88935-7b91-49ed-8336-0ddb10af5582.MOV
As seen the VoiceOver only detects the whole widget, not each subwidget (I try to click at all the lines and search box)

**Smartphone (please complete the following information):**
 - Device: iPhone 12 Pro
 - OS: 16.1.1
 - 5.0.5

**Additional context**
A possible solution might be to wrap the widgets with a Semantic widget.

**Comment by salim-lachdhaf** [comment]
check the v6.0.1 :)
